lib/src/wrappers/WrapperVariantContext.tsx

```diff
@@ -0,0 +1,5 @@
+import { createContext } from 'react';
+import { WrapperVariant } from './Wrapper';
+
+// consider getting rid from wrapper variant
```

Comment:
What alternative do you have in mind?

Reply:
Just passing through the props. But now I tent to left a context.

Reply:
Context sounds great :)